webextensions/manifest/manifest_version.json

```diff
@@ -18,7 +18,89 @@
               "version_added": "48"
             },
             "opera": {
-              "version_added": true
+              "version_added": "15"
+            }
+          }
+        },
+        "1": {
+          "__compat": {
+            "description": "Version 1",
+            "support": {
+              "chrome": {
+                "version_added": "4",
+                "version_removed": true
```

Comment:
needinfo?(@jpmedley): In which version did **Google&nbsp;Chrome** drop support for&nbsp;extension&nbsp;Manifest&nbsp;v1?